### PR TITLE
Restyled contact with form

### PR DESCRIPTION
**Pull request by FlemmingMP**
Draft of what the contact site looks like with Brief component and a form with inputs.

### CODE REVIEW

**Comment by FlemmingMP** [comment]
Old:
Old contact page

New:
New contact page

File: src/pages/kontakt/index.astro [hunk 1]
```diff
@@ -64,36 +65,53 @@ const employees = people.filter(person => person.data.email.includes('@bjerk.io'
   }}
 >
   <main>
-    <section class="brief">
-      <h1>Vi gleder oss til å høre fra deg!</h1>
+    <Brief title='Kontakt'>
+      <p>Vi gleder oss til å høre fra deg!</p>
       <p>
         Bjerk har kontorer i Oslo ved Akersgata 51, 0180 Oslo og Finland. Du får
```
Comment (Xillians) [inline src/pages/kontakt/index.astro:71]:
```suggestion
        Bjerk har kontorer i Oslo ved Arbins Gate 11, 0253 Oslo og Finland. Du får
```

File: src/pages/kontakt/index.astro [hunk 2]
```diff
@@ -64,36 +65,53 @@ const employees = people.filter(person => person.data.email.includes('@bjerk.io'
...
+          </div>
+          <div>
+            <dt>Adresse</dt>
+            <dd>
+              <a
+                href="https://goo.gl/maps/FkcqThLgRtCauPmZA"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Akersgata 51, 0180 Oslo <!-- <ArrowSquareOut /> -->
```
Comment (Xillians) [inline src/pages/kontakt/index.astro:89]:
```suggestion
                Arbins Gate 11, 0253 Oslo <!-- <ArrowSquareOut /> -->
```

**Comment by Xillians** [comment]
image
Draft v2

**Comment by Xillians** [comment]
image
V3

**Review by Xillians — APPROVED** [review]

**Pull request merged**